opencv/src/OpenCV/ImgProc/GeometricImgTransform.hsc

```diff
@@ -458,6 +459,26 @@ getPerspectiveTransform srcPts dstPts = unsafeCoerceMat $ unsafePerformIO $
             );
         }|]
 
+{- | Calculates an affine transformation matrix for 2D affine transform
+
+<http://docs.opencv.org/3.0-last-rst/modules/imgproc/doc/geometric_transformations.html#getaffinetransform OpenCV Sphinx doc>
+-}
+getAffineTransform
+    :: (IsPoint2 point2 CFloat)
+    => V.Vector (point2 CFloat) -- ^ Array of 3 floating-point Points representing vertices in source image
+    -> V.Vector (point2 CFloat) -- ^ Array of 3 floating-point Points representing vertices in destination image
+    -> Mat (ShapeT [2, 3]) ('S 1) ('S Double) -- ^ The output perspective transformation, 2x3 floating-point-matrix.
+getAffineTransform srcPts dstPts = unsafeCoerceMat $ unsafePerformIO $
```

Comment:
You have to catch potential C++ exceptions. Otherwise they will kill the haskell RTS. And `cv::getAffineTransform` _will_ throw exceptions. Use `unsafeWrapException` and the `cvExcept` quasiquoter.